crates/ruma-events/src/sticker.rs

```diff
@@ -95,16 +98,24 @@ pub struct StickerEventContent {
     /// The media source of the sticker image.
     #[serde(flatten)]
     pub source: StickerMediaSource,
+
+    /// Information about related messages.
+    #[serde(
+        flatten,
+        skip_serializing_if = "Option::is_none",
+        deserialize_with = "crate::room::message::relation_serde::deserialize_relation"
+    )]
+    pub relates_to: Option<Relation<RoomMessageEventContentWithoutRelation>>,
```

Comment:
The inner type is not correct here, it should be a `StickerEventContentWithoutRelation`. The `EventContent` macro can generate it automatically by adding `, without_relation` to the `ruma_event` attribute.

Reply:
Thank you for your instruction on the inner type, as for unit testing of ruma I'm not familiar but I'll try to write something